augur/tree.py

```diff
@@ -140,10 +141,10 @@ def build_iqtree(aln_file, out_file, substitution_model="GTR", clean_up=True, nt
           "\n\tMol. Biol. Evol., 32:268-274. https://doi.org/10.1093/molbev/msu300\n")
     if substitution_model.lower() == "none":
         print("Conducting a model test... see iqtree.log for the result. You can specify this with --substitution-model in future runs.")
-    os.system(cmd)
 
     # Check result
```

Comment:
This comment no longer makes sense as you've moved the system call after `# check result`.

Reply:
I removed the comment in 5c86faa.